src/main/java/training/supportbank/transactionhandlers/ProcessTransactions.java

```diff
@@ -22,12 +24,13 @@
             BigDecimal currentValue = BigDecimal.valueOf(0.00);
 
             for(Transaction transaction : transactions) {
-                if (transaction.from.equals(name)) {
-                    currentValue = currentValue.add(transaction.amount);
-                } else if (transaction.to.equals(name)) {
-                    currentValue = currentValue.subtract(transaction.amount);
+                if (transaction.getFrom().equals(name)) {
+                    currentValue = currentValue.add(transaction.getAmount());
+                    accountBalances.put(name, currentValue);
+                } else if (transaction.getTo().equals(name)) {
+                    currentValue = currentValue.subtract(transaction.getAmount());
+                    accountBalances.put(name, currentValue);
                 }
-                accountBalances.put(name, currentValue);
```

Comment:
Could we have a chat about this line? It came up in the last review, and I'm still not sure about it's position.